configs/rec/robustscanner/README.md

```diff
@@ -0,0 +1,387 @@
+
+<details open markdown>
+  <div align="center">
+  <summary>Detailed accuracy results for each benchmark dataset</summary>
+
+| **Model** | **Backbone** | **IIIT5k** | **SVT** | **IC13** | **IC15** | **SVTP** |**CUTE80** | **Average** |
+| :------: | :------: |:----------:|:-------:|:--------:|:--------:|:--------:| :------: |:---------:|
+| RobustScanner  | ResNet-31 |   95.23%   | 89.64%  |  93.50%  |  74.48%  |  81.55%  | 87.85%  |  87.51%   |
+  </div>
+</details>
+
+**Notes:**
+- Context: Training context denoted as {device}x{pieces}-{MS mode}, where mindspore mode can be G-graph mode or F-pynative mode with ms function. For example, D910x4-MS1.10-G is for training on 4 pieces of Ascend 910 NPU using graph mode based on Minspore version 1.10.
+- To reproduce the result on other contexts, please ensure the global batch size is the same.
+- The model uses an English character dictionary, en_dict90.txt, consisting of 90 characters including digits, common symbols, and upper and lower case English letters. More explanation on dictionary, please refer to [4. Character Dictionary](#4-character-dictionary).
+- The models are trained from scratch without any pre-training. For more dataset details of training and evaluation, please refer to [Dataset Download & Dataset Usage](#312-dataset-download) section.
+- The input Shapes of MindIR of RobustScanner is (1, 3, 48, 160) and it is for Ascend only.
+
+## 3. Quick Start
+### 3.1 Preparation
+
+#### 3.1.1 Installation
+Please refer to the [installation instruction](https://github.com/mindspore-lab/mindocr#installation) in MindOCR.
+
+#### 3.1.2 Dataset Download
+The dataset used for training and validation in this work, was referenced from the datasets used by mmocr and PaddleOCR for reproducing the RobustScanner algorithms. We are very grateful to mmocr and PaddleOCR for improving the reproducibility efficiency of this repository.
+
+The details of the dataset are as follows:
+<div align="center">
+
+| **Training set** |    **instance num**    | **repeat num** | **type** |
+|:----------------:|:--------------:|:--------:|:--------:|
+|    icdar2013     |      848       |    20    |   real   |
+|    icdar2015     |      4468      |    20    |   real   |
+|    coco_text     |     42142      |    20    |   real   |
+|      IIIT5K      |      2000      |    20    |   real   |
+|    SynthText     |    2400000     |    1     |  synth   |
+|     SynthAdd     |    1216889     |    1     |    synth    |
+|      Syn90k      |    2400000     |    1     |    synth    |
+
+</div>
+Note: SynthText and Syn90k datasets were both randomly sampled the complete SynthText dataset and complete Syn90k dataset,  consisting of 2.4 million samples each.
+
+The training and evaluation datasets in LMDB format shown in the table above can be downloaded from [here](https://download.mindspore.cn/dataset/robustscanner_dataset/). The downloaded file contains several compressed files, including:
```

Comment:
datasets uploaded to download server is a concern.

Reply:
Fixed, refer to AI Studio download url.